src/Germany/Mews.Fiscalizations.Germany/Mews.Fiscalizations.Germany.csproj

```diff
@@ -10,7 +10,7 @@
     <RepositoryUrl>https://github.com/MewsSystems/fiscalizations</RepositoryUrl>
     <Icon>https://raw.githubusercontent.com/msigut/eet/master/receipt.png</Icon>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <PackageVersion>4.0.0</PackageVersion>
+    <PackageVersion>4.0.1</PackageVersion>
```

Comment:
According to the [semantic versioning](https://semver.org/), we should bump the minor version when we extend the public interface like this.